app/controllers/homepage_controller.rb

```diff
@@ -59,11 +59,14 @@ def index
     shape_name_map = all_shapes.map { |s| [s[:id], s[:name]]}.to_h
 
     if @view_type == 'map'
-      viewport = if params[:boundingbox]
-        sw_lat, sw_lng, ne_lat, ne_lng = params[:boundingbox].split(",")
-        { sw: [sw_lat, sw_lng], ne: [ne_lat, ne_lng]}
+      coords = Maybe(params[:boundingbox]).split(',').or_else
```

Comment:
Exactly that, returns `nil`. I agree that this sounds a bit ominous, I'll have a look around what's the convention mostly used in our code.